src/core/cli/initializer.ts

```diff
@@ -144,6 +145,10 @@ class Initializer {
       await this.initEnv();
       conf.browserWidth = parseInt(await this.promptFolders('What is desired browser width?', conf.browserWidth));
       conf.browserHeight = parseInt(await this.promptFolders('What is desired browser height?', conf.browserHeight));
+      conf.browserLanguage = await this.promptFolders(
```

Comment:
what will happen if i don't enter anything ?

Reply:
Default will be used - en-GB

Reply:
Ah i see didn't noticed.

Reply:
I've just checked how it works. If the user would set the browserLanguage in kakunin.conf.js into empty string/null/undefined. Then the default language from our system would be used (system preferences -> language & region -> preferred languages).

I will add if for that, and the "en-GB" would be always set if someone would break that value.